module-1/lab-mysql-select/your-code/solutions.sql

```diff
@@ -0,0 +1,45 @@
+#challenge 1
+
+SELECT authors.au_id AS 'AUTHOR ID', authors.au_lname AS 'LAST NAME', authors.au_fname AS 'FIRST NAME', titles.title AS TITLE, publishers.pub_name AS PUBLISHER
+FROM authors, titles, publishers
+WHERE title_id = title_id
+
+#challenge 2
+
+SELECT 	authors.au_id AS 'AUTHOR ID', 
+		authors.au_lname AS 'LAST NAME', 
+        authors.au_fname AS 'FIRST NAME', 
+        publishers.pub_name AS 'PUBLISHER',
+        COUNT(titles.title_id) AS 'TITLE COUNT'
+FROM authors
+LEFT JOIN titleauthor
+ON titleauthor.au_id=authors.au_id
+LEFT JOIN titles
+ON titles.title_id=titleauthor.title_id 
+LEFT JOIN publishers
+ON titles.pub_id=publishers.pub_id
+GROUP BY authors.au_id, publishers.pub_name;
+
```

Comment:
En esta challenge es mejor utlizar un inner join, in vez que un left join, porque van a salir author con publisher `null` y con `count = 0`